### DISCUSSION TITLE
[feature request] Lists of hashtags instead of users

### DISCUSSION THREAD

**Discussion by ismailghedamsi** [mastodon/mastodon#21772] [General]
One of my main issues right now with mastodon is that there's no way to separate toots by topics. It would be cool to have a list that allows grouping hashtags in lists instead of users.

**Comment by shawnhcorey** [comment]
Yeah, that would be great. For example, I'm interested in the show Star Trek: Prodigy. It has two hashtags #StarTrekProdigy and #Prodigy. Putting them in a list would mean I don't have to do two searches.

**Comment by Truhe2342** [comment]
Isn't that already implemented?
![image](https://user-images.githubusercontent.com/118095316/210563694-6822691a-6613-444e-8261-a92ab112ffdd.png)

**Reply by btaroli** [reply]
No, this isn't about following a hashtag. It's about adding a hashtag to a LIST. And I definitely agree this is a good thing. For hashtags, like some users, that are very chatty, i find it's good to have them in a list I can check on periodically rather than wade through their toots in my timeline.

See https://github.com/mastodon/mastodon/issues/22812

**Comment by bcmike70** [comment]
Agreed this would be an excellent feature. My home timeline is super busy so being able to break out followed hash tags would be spectacular!

**Comment by johnmcn1** [comment]
I've had this same thought ever since v4 got released.  I'm convinced it would super-charge the use of hashtags.

**Comment by loremipsum2022** [comment]
See #7692 - it would be even better to be able to create mixed lists containing hashtags and accounts.

**Comment by nileshtrivedi** [comment]
I submitted this feature request: https://github.com/mastodon/mastodon/issues/31505

**Comment by nileshtrivedi** [comment]
I submitted this [initial PR](https://github.com/mastodon/mastodon/pull/31549) to add a boolean column `highlighted` to tags table.
This is for enabling instance-level "featured" tags (think the topics/channels sidebar in Slack or Discord), and not for a user to add hashtags to their personal list.

**Comment by nileshtrivedi** [comment]
Is there any ActivityPub server implementation which supports this feature (users being able to add hashtags to a list)? That would make it easier to start adding support for this in various clients (such as Whalebird) without being blocked on Mastodon devs.
